Tests/UnitTests/Purchasing/Purchases/PurchasesLogInTests.swift

```diff
@@ -113,6 +113,30 @@ class PurchasesLogInTests: BasePurchasesTests {
         expect(self.identityManager.invokedSwitchUserParametersList) == ["test-user-id"]
     }
 
+    func testRefreshesOfferingsCache() {
+        expect(self.mockOfferingsManager.invokedUpdateOfferingsCacheCount) == 0
+
+        self.systemInfo = MockSystemInfo(finishTransactions: true, customEntitlementsComputation: true)
```

Comment:
This initialization is duplicated in several tests now, might be worth extracting.